taskRequests/script.js

```diff
@@ -84,6 +92,174 @@ function openTaskDetails(id) {
   window.location.href = url;
 }
 
+if (isDev) {
+  containerFilter.style.display = 'flex';
+  oldContainerFilter.style.display = 'none';
+} else {
+  containerFilter.style.display = 'none';
+  oldContainerFilter.style.display = 'flex';
+}
+sortButton.addEventListener('click', async (event) => {
+  event.stopPropagation();
+  sortModal.classList.toggle('hidden');
+  sortModalButtons();
+});
+
+window.onclick = function () {
+  sortModal.classList.add('hidden');
+  filterModal.classList.add('hidden');
+};
```

Comment:
Can we use `window.addEventListener` here?

Reply:
implemented backdrop instead of window